src/lib/src/charts/timeseries-chart.component.ts

```diff
@@ -97,28 +81,69 @@ export class TimeseriesChartComponent implements OnChanges {
       }
 
       if (this.formatX) {
-        config.axis.x['type'] = 'timeseries';
-        config.axis.x.tick['format'] = this.formatX;
+        config['axis'] = {
+          ...config['axis']
```

Comment:
Is this necessary since we're not merging any new values at the top level?

Reply:
yeah I think that's a good call. will fix!